sklearn/linear_model/tests/test_sag.py

```diff
@@ -659,21 +656,22 @@ def test_sag_multiclass_computed_correctly(csr_container):
     """tests if the multiclass classifier is computed correctly"""
     alpha = 0.1
     n_samples = 20
-    tol = 0.00001
-    max_iter = 40
+    tol = 1e-5
+    max_iter = 70
```

Comment:
I assume this was needed because the previous `max_iter` value was too RNG sensitive?

Reply:
Yes. I checked the new (and old) value with all global random seeds. SAG and SAGA work better for larger datasets,